### PR TITLE
Update managing-roles-and-permissions.md

### PR DESCRIPTION
**Pull request by SpiritLevel**
Minor typos and rewording.

### CODE REVIEW

File: docs/en/for-website-administrators/managing-roles-and-permissions.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ Learn how to restrict access by IP address
...
 
 Roles are collections of permissions.  For example, you might create an editor role to give a group read/write access to all content, or a "contributor" role who has the right to add content to the CMS but not to publish it, or a "spectator" role which gives a person a right to view the backend of the CMS, but not any ability to edit it.
 
-Security groups are collections of users, and whatever permissions they have apply to a subset of pages. So, for example, the marketing team could have access to the parts of the website dealing with marketing, and the development team could have access to the parts of the website dealing with development.
+Security groups are collections of users. Whatever permissions each user in a group has, these permissions apply to pages accessible by the group. So, for example, the marketing team could have access to the parts of the website dealing with marketing (some as editors, some as contributors, and some as spectators), and the development team could have access to the parts of the website dealing with development (some as editors, some as contributors, and some as spectators).
```
Comment (chillu) [inline docs/en/for-website-administrators/managing-roles-and-permissions.md]: "spectators" seems like unusual terminology in this context. Maybe "reviewers" fits better here?

Comment (SpiritLevel) [inline docs/en/for-website-administrators/managing-roles-and-permissions.md]: Absolutely :) Although, some reviewers don't do much of anything and should be considered passive spectators! Changed.

Comment (chillu) [inline docs/en/for-website-administrators/managing-roles-and-permissions.md]: Did you forget to push the change? ;)

Comment (SpiritLevel) [inline docs/en/for-website-administrators/managing-roles-and-permissions.md]: I think I did...still getting used to Github....

**Pull request merged**